app/routes/datasets/downloads.py

```diff
@@ -194,6 +197,78 @@ async def download_csv_post(
     return response
 
 
+@router.get(
+    "/{dataset}/{version}/download_by_aoi/csv",
+    response_class=CSVStreamingResponse,
+    tags=["Download"],
+)
+async def download_by_aoi_csv(
+    dataset_version: Tuple[str, str] = Depends(dataset_version_dependency),
+    sql: str = Query(..., description="SQL query."),
+    aoi: AreaOfInterest = Depends(_parse_area_of_interest),
+    filename: str = Query("export.json", description="Name of export file."),
+    delimiter: Delimiters = Query(
+        Delimiters.comma, description="Delimiter to use for CSV file."
+    ),
+):
+    """Execute a READ-ONLY SQL query on the given dataset version (if
+    implemented) for a given AOI.
+
+    Returns downloads in either JSON or CSV depending on the Accept
+    header.
```

Comment:
I don't think this is true for these two endpoints. We've kept the format as part of the path.